build/wd_ts_project.bzl

```diff
@@ -1,15 +1,41 @@
 load("@aspect_rules_ts//ts:defs.bzl", "ts_project")
+load("@npm//:eslint/package_json.bzl", eslint_bin = "bin")
 
-def wd_ts_project(name, srcs, deps, testonly = False):
+def wd_ts_project(name, srcs, deps, data = [], eslintrc_json = None, testonly = False):
     """Bazel rule for a workerd TypeScript project, setting common options"""
 
+    tsconfig = "//types:tsconfig.json"
+
     ts_project(
         name = name,
         srcs = srcs,
         deps = deps,
-        tsconfig = "//types:tsconfig.json",
+        tsconfig = tsconfig,
         allow_js = True,
         composite = True,
         source_map = True,
         testonly = testonly,
     )
+
+    if eslintrc_json:
+        eslint_bin.eslint_test(
```

Comment:
We'd want this to consolidate this with the other places where we call eslint_test() to factor them all out into one function, but that's not needed for now

Reply:
Agreed. I can follow up in a different pull-request.

Reply:
I went ahead and implemented this on a new commit. Would you mind taking a look @fhanau?

Reply:
Thank you for doing so!

Couple smaller things:
- Load statements in 1st line of eslint_test.bzl are unused, can be dropped (if you use the VSCode Bazel extension it will warn about stuff like this automatically)
- `//:prettierrc` is no longer being passed in the data argument – I assume it was unused before?
- Ideally we'd also fold the `eslint_bin.eslint_test` target in `src/wpt/BUILD.bazel` into this (haven't checked if there'd be additional complexity with it)

Reply:
> //:prettierrc is no longer being passed in the data argument – I assume it was unused before?

Yes prettier is not used at all.

> Ideally we'd also fold the eslint_bin.eslint_test target in src/wpt/BUILD.bazel into this (haven't checked if there'd be additional complexity with it)

Yes, I did another take. Nice catch.

Reply:
LGTM.